eoscompanion/main.py

```diff
@@ -149,17 +149,79 @@ def do_dbus_register(self, connection, object_path):  # pylint: disable=argument
 
     def do_dbus_unregister(self, connection, object_path):  # pylint: disable=arguments-differ
         '''Invoked when we lose a D-Bus connection.'''
-        log('Lost session d-bus connection at {path}'.format(path=object_path))
+        logging.warning('Lost session d-bus connection at %s', object_path)
         return Gio.Application.do_dbus_unregister(self,
                                                   connection,
                                                   object_path)
 
     def do_activate(self):  # pylint: disable=arguments-differ
         '''Invoked when the application is activated.'''
-        log('Activated')
+        logging.info('Activated')
         return Gio.Application.do_activate(self)
 
 
+COMPANION_APP_CONFIG_FILES = [
+    '/var/run/host/etc/eos-companion-app/config.ini',
+    '/var/run/host/usr/share/eos-companion-app/config.ini',
+    # FIXME: /var currently does not bind in /run/host in the sandbox
```

Comment:
I'm pretty sure that you can explicitly mount `/var/lib/eos-companion-app` from the manifest...
Also, the `/var` path should take priority over the `/usr` one so it should be moved one line up